config/settings.yml

```
@@ -392,3 +392,14 @@ bing:
 flipper:
   username: SUPER_SECRET_USERNAME
   password: SUPER_SECRET_PASSWORD
+
+oit_lighthouse2:
+  sql52:
+    hostname: vhacdwdwhsql52.vha.med.va.gov
+    username: fake_username
+    password: fake_password
+    port: 1433
+    facilities_mental_health:
+      database_name: OIT_Lighthouse2
+      schema_name: A01
+      table_name: OMHSP_PERC_Share__DOEx__FieldDataEntry_MHPhone
```

```suggestion
      table_name: VHA_Mental_Health_Contact_Info
```
This is my proposed name for the view created for the app utility account to query (it is filled with the data in the table `OMHSP_PERC_Share__DOEx__FieldDataEntry_MHPhone`, but I thought we might give the view a more readable name.